master/internal/config.go

```diff
@@ -87,6 +87,10 @@ type Config struct {
 	HPImportance          hpimportance.HPImportanceConfig   `json:"hyperparameter_importance"`
 
 	*resourcemanagers.ResourceConfig
+
+	// Below here are "hidden" useful debugging configurations.
```

Comment:
Maybe break out an `internal` block or something to hold this sort of thing with a bit more structure?